pages/orders/[oid].tsx

```diff
@@ -0,0 +1,74 @@
+import { Button, Flex, H3, Panel, Text } from '@bigcommerce/big-design';
+import { useRouter } from 'next/router';
+import ErrorMessage from '../../components/error';
+import Loading from '../../components/loading';
+import { useOrderInfo } from '../../lib/hooks';
+
+const OrderInfo = () => {
+    const router = useRouter();
+    const oid = Number(router.query?.oid);
+    const { isLoading, order, error } = useOrderInfo(oid);
+
+    const {
+        billing_address: {
+            city = '',
+            first_name: firstName = '',
+            last_name: lastName = '',
+            state = '',
+            street_1: street1 = '',
+            street_2: street2 = '',
+            zip = '',
+            country = '',
+        } = {},
+        id = 0,
+        items_total: itemsTotal = 0,
+        order_source: orderSource = '',
+        payment_status: paymentStatus = '',
+        status: orderStatus = '',
+        total_inc_tax: totalIncTax = '',
+     } = order ?? {};
+
+    const formatCurrency = (amount: string) => {
+        return amount.replace(amount.substring(amount.length - 3, amount.length -1), '');
+    }
+
+    const handleClick = () => {
+      router.back();
+    }
+
+    if (isLoading) return <Loading />;
+    if (error) return <ErrorMessage error={error} />;
+
+    return (
+      <>
+          <Panel header={`Order - #${id}`}>
+              <Flex justifyContent="flex-start" marginTop="large">
+                  <Flex flexDirection="column" alignItems="start" marginRight="xxxLarge">
+                    <H3>Customer</H3>
+                    <Text>{`${firstName} ${lastName}`}</Text>
```

Comment:
note: we don't need to wrap in a template literal, we can just specify the variables
```suggestion
                    <Text>{firstName} {lastName}</Text>
```